src/components/Modal/index.js

```diff
@@ -65,46 +66,55 @@ const CancelButtonContainer = styled(Row)`
   padding-right: ${spacing.moderate};
 `;
 
-const Modal = ({ children }) => (
+const Modal = ({ children, container, content, ...props }) => (
   <BackdropConsumer>
     {backdropValue => (
       <ModalContainer
         small={4}
         role="dialog"
         aria-modal
         innerRef={backdropValue ? backdropValue.childRef : null}
+        style={{ ...container }}
```

Comment:
Rather than using the `container` and `content` props as values for the `ModalContainer` and `ModalContent` components, respectively, can you please rename the props to `containerProps` and `contentProps` and spread them into the components? That way we have greater flexibility to pass props to these components moving forward.

Reply:
nice!